util/lambda/index.js

```diff
@@ -0,0 +1,318 @@
+/* eslint-disable no-useless-catch */
+/* eslint-disable no-unused-vars */
+const JSZip = require("jszip");
+const fs = require("fs");
+const {
+  LambdaClient,
+  CreateFunctionCommand,
+  GetFunctionCommand,
+  InvokeCommand,
+  UpdateFunctionCodeCommand,
+  waitUntilFunctionActiveV2,
+  waitUntilFunctionUpdatedV2
+} = require("@aws-sdk/client-lambda");
+const logger = require("../../logger");
+const stats = require("../stats");
+const {
+  isABufferValue,
+  bufferToString,
+  TRANSFORM_WRAPPER_CODE
+} = require("./utils");
+
+const MAX_WAIT_TIME = parseInt(process.env.MAX_WAIT_TIME || "30", 10);
+const DELAY = parseInt(process.env.DELAY || "2", 10);
+const EVENT_SIZE_LIMIT_IN_MB =
+  parseInt(process.env.EVENT_SIZE_LIMIT_IN_MB || "3", 10) * 1024 * 1024;
+
+const client = new LambdaClient({
+  region: process.env.AWS_REGION || "us-east-1"
+});
+
+const waiterConfig = {
+  client,
+  maxWaitTime: MAX_WAIT_TIME,
+  minDelay: DELAY,
+  maxDelay: DELAY
+};
+
+const createZip = async (fileName, code) => {
```

Comment:
(fileName, code) -> (fileName, userCode)

Reply:
should this return string contents of fs.readFile too as we are handling it outside and multiple times

Reply:
done